packages/shared/components/modals/ProfileActions.svelte

```diff
@@ -47,6 +48,15 @@
             })
         }
     }
+
+    let isCheckingLedger: boolean = false
```

Comment:
Anotating a boolean type when assigning a boolean value is redundant. Type annotations are only needed when the TS compiler cannot figure out on its own the type